src/webgpu/api/operation/queue/writeBuffer.spec.ts

```diff
@@ -8,3 +8,89 @@ import { makeTestGroup } from '../../../../common/framework/test_group.js';
 import { GPUTest } from '../../../gpu_test.js';
 
 export const g = makeTestGroup(GPUTest);
+
+g.test('writeBuffer,Ranges')
```

Comment:
Needs a `.desc()` with a test plan that explains what the test tests in a bit more detail. BTW, have you read https://github.com/gpuweb/cts/blob/main/docs/intro/README.md?